docs/news.html

```diff
@@ -7,15 +7,15 @@
   </head>
   <body>
     <div class="news-item">
-      <span class="item-date">2025/11/03</span>
-      <a href="https://forum.getodk.org/t/57029" target="_blank">
-        ODK Central v2025.3.0
+      <span class="item-date">2025/11/13</span>
+      <a href="https://forum.getodk.org/t/57029/2" target="_blank">
```

Comment:
```suggestion
      <a href="https://forum.getodk.org/t/57029/4" target="_blank">
```

I think Yaw pinning and unpinning the post count as /2 and /3.